Cargo.lock

```diff
@@ -859,7 +909,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fd94671561e36e4e7de75f753f577edafb0e7c05d6e4547229fdf7938fbcd2c3"
 dependencies = [
  "core2",
- "multihash 0.18.0",
+ "multihash 0.18.1",
```

Comment:
This is the crate version that fixes the panic.

Reply:
I suggest bumping the dependency in `Cargo.toml` as well.

Reply:
This will technically work, but it might be nice to submit a PR to https://github.com/multiformats/rust-cid/blob/master/Cargo.toml to bump the actual dependency there.

Reply:
Then we could bump our `cid` dependency in `Cargo.toml`.
Meanwhile, approving--but maybe file a chore to follow up once the `cid` crate bumps the underlying dependency.

Reply:
@saraswatpuneet , it's not a manual update of the lock file; I think Cargo updated the indirect dependency based on the rules, now that multihash has published a version with the fix. But my previous comments stand--we should encourage the `cid` crate to update their dependency, and then update ours explicitly.